src/workerd/api/streams/standard.c++

```diff
@@ -3353,6 +3355,10 @@ jsg::Ref<WritableStream> WritableStreamJsController::addRef() {
   return KJ_ASSERT_NONNULL(owner).addRef();
 }
 
+bool WritableStreamJsController::isClosedOrClosing() {
+  return state.is<StreamStates::Closed>();
```

Comment:
An errored state would also imply that the stream is closed.

Reply:
I mirrored WritableStreamInternalController::isClosedOrClosing. Perhaps things are different for WritableStreamJsController but then maybe I should just make this KJ_UNIMPLEMENTED.

Reply:
If it's not being used at all for anything, just make it `KJ_UNIMPLEMENTED`